filo-cli

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+SCALA_VERSION="2.10"
+FILO_VERSION="0.1-SNAPSHOT"
```

Comment:
We should split out the version into a version.sbt, then it could be easily grep'ed.  For now this might be ok but we do need a more automatic way of getting the version.